modules/ROOT/pages/alerts-us-eu.adoc

```diff
@@ -0,0 +1,92 @@
+= Getting Alerts (US and EU Clouds)
```

Comment:
NO NEED TO REVIEW, as this only has a title tweak. It's new in the PR only because of the file name change. File with old name (alerts.adoc) is now the landing page for the different instances of Anypoint Platform.